gradle/libs.versions.toml

```diff
@@ -10,7 +10,7 @@ junit-jupiter = "5.11.2"
 kotlin = "2.1.10"
 kotlin-compile-testing = "0.7.0"
 kotlin-hierarchy = "1.1"
-kotlin-inject = "0.7.2"
+kotlin-inject = "0.7.3-SNAPSHOT"
```

Comment:
- [ ] revert back to stable version since assisted factory annotation wasn't needed